core/src/processing/opengl/PSurfaceJOGL.java

```diff
@@ -195,7 +195,8 @@ protected void initGL() {
     if (profile == null) {
       if (PJOGL.profile == 2) {
         try {
-          profile = GLProfile.getGL2ES1();
+          // request at least GL2 or GLES2
+          profile = GLProfile.getGL2ES2();
```

Comment:
Great… will update and send a new PR.
On Aug 11, 2015 21:10, "Ben Fry" <notifications@github.com> wrote:

> In core/src/processing/opengl/PSurfaceJOGL.java
> <https://github.com/processing/processing/pull/3566#discussion_r36785725>:
>
> > @@ -195,7 +195,8 @@ protected void initGL() {
> >      if (profile == null) {
> >        if (PJOGL.profile == 2) {
> >          try {
> > -          profile = GLProfile.getGL2ES1();
> > +          // request at least GL2 or GLES2
> > +          profile = GLProfile.getGL2ES2();
>
> But it sounds like that might take a little while, so let's add a platform
> check there and get the ES2 profile if it's running on ARM, and we can use
> that for the time being until #3505
> <https://github.com/processing/processing/issues/3505> is finished.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/processing/processing/pull/3566/files#r36785725>.
>